common/persistence/dataManagerInterfaces.go

```diff
@@ -610,6 +618,13 @@ type (
 		TargetCluster string
 	}
 
+	// CrossClusterApplyParentClosePolicyTask is the cross-cluster version of ApplyParentClosePolicyTask
+	CrossClusterApplyParentClosePolicyTask struct {
+		ApplyParentClosePolicyTask
```

Comment:
We can actually flatten this struct and only use `CrossClusterApplyParentClosePolicyTask`; not sure if we wanted to keep the convention. Let me know what you think.